smartseq2_single_sample/tasks/hisat2_pipeline.wdl

```diff
@@ -0,0 +1,101 @@
+import "hisat2.wdl" as hisat2
+import "picard.wdl" as picard
+import "featurecounts.wdl"  as featurecounts
+import "htseq.wdl" as htseq
+
+workflow RunHisat2Pipeline {
+  File fastq_read1
+  File fastq_read2
+  File gtf
+  File stranded
```

Comment:
Aren't your workflows always going to have the same strandedness? Could this be passed as a default argument?

Reply:
for now, workflows only take care unstranded. just want to keep some options for future changes.

Reply:
Ok.